_posts/2020-02-06-how-to-configure-a-production-grade-ci-cd-setup-for-apps-and-infrastructure-code.adoc

```diff
@@ -1350,15 +1394,12 @@ invoker. We will deploy both using the
 https://github.com/gruntwork-io/module-ci/tree/master/modules/ecs-deploy-runner[ecs-deploy-runner module] in
 `module-ci`.
 
-Add the Terraform files for the `ecs-deploy-runner` in `infrastructure-modules`:
+Create a new module called `ecs-deploy-runner` in `infrastructure-modules`:
```

Comment:
Soon customers will be able to just use the service-catalog module! Are you expecting to update this again once released?

Reply:
Yes.